packages/fast-css-editor-react/src/spacing/README.md

```diff
@@ -0,0 +1,36 @@
+# CSSSpacing
+The `CSSSpacing` component shows the CSS spacing (margin & padding) value as four input elements, it will also provide a callback for updating this values. When the `spacingType` is set to "margin" the inputs will adjust the top/bottom/left/right margins and when `spacingType` is set to "padding" it will adjust the top/bottom/left/right padding. The components `spacingType` will default to "margin" if not set.
```

Comment:
Doesn't feel like "value as four input elements" is accurate?  Are these really "elements"?  Aren't they styling "attributes" ?

Reply:
They are input HTML elements, this is essentially a component made up of form elements that are passing back CSS properties and values, which is what all of these components are doing.